nunjucks/src/object.js

```diff
@@ -59,4 +60,26 @@ class Obj {
   }
 }
 
-module.exports = Obj;
+class EmitterObj extends EventEmitter {
```

Comment:
Probably not worth changing now, but other EventEmitters are available, e.g. [little-emitter](https://www.npmjs.com/package/little-emitter), which is smaller and doesn't require the addition or modification of a superclass e.g. :

```js
const EventEmitter = require('little-emitter'); 

class Environment {
    constructor() {
        EventEmitter(this); // mix in `emit`, `on` etc.
    }
}
```

Reply:
I went with the default webpack events shim just because it's so widely used, it was easy, it's well documented, and it only added 1.5K gzipped to the minified builds.